imports/pages/groups/profile/index.js

```diff
@@ -37,12 +37,55 @@ export const JoinWithPhones = graphql(PHONE_QUERY, {
   }),
 })(Join);
 
+export const PHONE_NUMBER_MUTATION = gql`
+  mutation SetPhoneNumber($phoneNumber: String!) {
+    setPhoneNumber(phoneNumber: $phoneNumber) {
+      error
+      success
+      code
+    }
+  }
+`;
+
+const withAddPhoneNumber = graphql(PHONE_NUMBER_MUTATION, {
+  props: ({ mutate }) => ({
+    addPhone: (phoneNumber) => mutate({
+      variables: { phoneNumber },
+    }),
```

Comment:
lets add and updateQueries return on this and update the query pulling in phone numbers 

http://dev.apollodata.com/react/cache-updates.html#updateQueries